.github/workflows/run-tests.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
       - name: Parse Node version
         id: parse-node-version
-        run: echo "##[set-output name=nvmrc;]$(cat .nvmrc)" 
```

Comment:
Hmm, I wonder why this step had the `##`s here previously? @Rkotyank I'd try undoing this change and seeing if the other two below are ok. Then we can narrow down the problem.

Reply:
Yes, did notice the ##. I will undo the change